README.md

```diff
@@ -1,2 +1,21 @@
 # korerorero-mouth-shapes
+
 Exposing DanielSWolf / rhubarb-lip-sync as a service
+
+## Installation
```

Comment:
Can you please mention the CMake dependency

Reply:
Done